template/bootstrap3/bin-checkpoint-delete-profile-component.html

```diff
@@ -2,10 +2,10 @@
     <div class="alert alert-warning" role="alert">
         <button type="button" class="btn btn-danger pull-right" ng-disabled="$ctrl.working"
                 i18n code="shop.account.deletion.confirmation.label" read-only
-                ng-click-confirm="$ctrl.submit();" confirm-message="{{::var}}" ng-if="::var">
+                ng-click-confirm="$ctrl.submit();" confirm-message="{{::var}}">
```

Comment:
Alright so I agree you removed the ng-if as you specified in the comment to this commit.
But why did you do this? I mean I can guess but I think it's better if the commit message explains what you fixed rather than simply states what you did. I can see what you did in the diff so keep the commit message to the "why".